src/radio-button.jsx

```diff
@@ -11,8 +11,9 @@ const RadioButton = React.createClass({
 
   propTypes: {
     /**
-     * Checked if true.
+     * Used internally by `RadioButtonGroup`.
      */
+    /* Checked if true. */
```

Comment:
We have two comments here. Only the first one if picked up by react-docgen. What is the goal?